### ISSUE TITLE
User input isn't sanitized on server while editing server names. Sanitization is done only for create versions

### ISSUE THREAD

**Issue by surajrane42** [ToolJet/ToolJet#4486]
### What is the expected behaviour?
User input should be sanitized while editing version names as it is done currently while creating versions

### What is the current behaviour?
Currently user input is not sanitized while editing version name

### How to reproduce the issue?

1. Edit any application
2. Go to versions dropdown & click on edit icon for editing the name
3. Now add any special character like '<' or '>' and save
4. The input is not sanitized on server & same value is returned from API on get call for versions

### Screenshots or Screencast
Version PUT API Payload
![VersionPutPayload](https://user-images.githubusercontent.com/58817684/196193130-b9827e50-dabb-4017-af01-46b17c15d723.png)

Version GET API response
![VersionGetResponse](https://user-images.githubusercontent.com/58817684/196193220-47e08970-a149-4689-9352-80b41c8d6fc9.png)

### Any possible solutions?
Possible solution is to sanitize the user input on backend as it is done for create version scenario

### If the bug is confirmed, would you be willing to submit a PR?
Yes

**Comment by akshaysasidrn** [comment]
@surajrane42 Yeah, sanitisation is missed on the version update. Would you help to pick it up and fix it?

**Comment by surajrane42** [comment]
@akshaysasidrn Sure. You can assign this to me
